src/main/java/com/meta/chatbridge/llm/HuggingFaceLlamaPlugin.java

```diff
@@ -135,51 +78,52 @@ public T handle(MessageStack<T> messageStack) throws IOException {
 
         JsonNode responseBody = MAPPER.readTree(response.returnContent().asBytes());
         String allGeneratedText = responseBody.get(0).get("generated_text").textValue();
-        String llmResponse = allGeneratedText.trim().replace(prompt.trim(), "");
+        String llmResponse = allGeneratedText.strip().replace(prompt.strip(), "");
         Instant timestamp = Instant.now();
 
         return messageStack.newMessageFromBot(timestamp, llmResponse);
     }
 
-    private String getPrompt(MessageStack<T> MessageStack) {
-        List<String> texts = new ArrayList<>();
+    public String createPrompt(MessageStack<T> MessageStack) {
```

Comment:
you should be able to make this package-private for testing (just remove the public)